tmp.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+from delira.data_loading import AbstractDataset, BaseDataManager
+
+
+class DummyDataset(AbstractDataset):
```

Comment:
Isn't there a better place and name than to call it `tmp.py`(do we need this file)?

Reply:
No we don't :D I already deleted it, this was pushed accidentally an was a debug script for me